scripts/test_wrapper

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-set -o pipefail
-
 RESULTS_FILE=$1
 TARGET_NAME=${@: -1}
 shift
 
 # Run remain args as test command, capturing the output.
-LINTER_OUTPUT=$( $@ 2>&1 | tee /dev/stderr )
+LINTER_OUTPUT=$( $@ 2>&1 )
```

Comment:
```suggestion
LINTER_OUTPUT="$( $@ 2>&1 )"
```

Reply:
fair enough. I initially matched what this script was originally doing: https://github.com/ros/roslint/blob/78c58b25ac4af3402d5827e6b6c8281366a8f29b/scripts/test_wrapper#L8